### PR TITLE
Add solution and tests for Lab 09

### PR DESCRIPTION
**Pull request by ivanangelov**

### CODE REVIEW

File: 09-threads/lab/solution/src/bg/sofia/uni/fmi/mjt/restaurant/customer/Customer.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	public Customer(Restaurant workshop) {
+		super(workshop);
+	}
+
+	@Override
+	public boolean hasVipCard() {
+		return false;
+	}
+
+}
```
Comment (100yo) [inline 09-threads/lab/solution/src/bg/sofia/uni/fmi/mjt/restaurant/customer/Customer.java:16]: Add an empty last line

File: 11-http-rest/lab/README.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
+# HTTP, REST, JSON и HttpClient
```
Comment (100yo) [inline 11-http-rest/lab/README.md:1]: Why this file is part of this pull request?

File: 09-threads/lab/solution/src/bg/sofia/uni/fmi/mjt/restaurant/Chef.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
+package bg.sofia.uni.fmi.mjt.restaurant;
+
+public class Chef extends Thread {
+
+	private final int id;
```
Comment (100yo) [inline 09-threads/lab/solution/src/bg/sofia/uni/fmi/mjt/restaurant/Chef.java:5]: Please fix indentation of all classes to 4 spaces.

**Review by 100yo — CHANGES_REQUESTED** [review]
Please review & fix review comments before merging.

File: 09-threads/lab/solution/src/bg/sofia/uni/fmi/mjt/restaurant/customer/Customer.java (hunk 1 above)
Comment (ivanangelov) [reply 09-threads/lab/solution/src/bg/sofia/uni/fmi/mjt/restaurant/customer/Customer.java]: Done.

File: 09-threads/lab/solution/src/bg/sofia/uni/fmi/mjt/restaurant/Chef.java (hunk 3 above)
Comment (ivanangelov) [reply 09-threads/lab/solution/src/bg/sofia/uni/fmi/mjt/restaurant/Chef.java]: Done.

File: 11-http-rest/lab/README.md (hunk 2 above)
Comment (ivanangelov) [reply 11-http-rest/lab/README.md]: Misplaced, removed it.

**Review by 100yo — APPROVED** [review]
Thanks for contributing and fixing review comments. Good to merge.

**Pull request merged**